stable/keel/templates/secrets-webhookrelay.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if .Values.webhookRelay.enabled }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ template "name" . }}-webhookrelay
```

Comment:
Should use `{{ template "fullname" . }}`. Actually, the additional suffix is not necessary. Update references accordingly.